app/components/CartTile/CartTile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import _ from 'lodash';
+import './_CartTile.scss';
+import MovieStore from '../../stores/MovieStore';
+import Rating from '../Rating/Rating';
+
+
+const findMovieFromAll = (props) => {
+	let title = props.cartItem.params.title;
+	return MovieStore.getByTitle(title);
+}
+
+let CartTile = (props) => {
+	let movie = findMovieFromAll(props) || {};
+	return (
+		<li className="cart-tile">
+			<div className="cart-container">
+				<div className="image-container"><img height='110px' src={'img/' + movie.cover} width='67px'/></div>
+				<div className="content-container bold">
+					<h2>{movie.title} {'('+movie.year+')'}</h2>
+					<p>{movie.description}</p>
+					<Rating stars={movie.rating ? Number(movie.rating) : 0} title={movie.title}/>
+				</div>
+			</div>
+		</li>
+	);
```

Comment:
I think it might be a tab vs space thing. Might need to just reindent a few files